src/cloudabi-run/cloudabi-run.cc

```diff
@@ -55,7 +55,9 @@ namespace {
 int main(int argc, char *argv[]) {
   // Parse command line options.
   bool do_emulate = false;
+  int execfd = -1;
```

Comment:
CloudABI's C/C++ style is to declare variables upon first use. Just write `int execfd = open(...)` further below.